### PR TITLE
added tags

### PR DESCRIPTION
**Pull request by cpareek**
### Change description ###
Added missing tags

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: apps/azureserviceoperator-system/resources/flexibleserver-postgres.yaml [hunk 1]
```diff
@@ -6,6 +6,12 @@ metadata:
   annotations:
     serviceoperator.azure.com/reconcile-policy: detach-on-delete
 spec:
+  tags:
+    builtFrom: https://github.com/hmcts/cnp-flux-config
+    businessArea: CFT
+    environment: development
+    autoShutdown: "true"
+    application: social-service-child-support
```
Comment (timja) [inline apps/azureserviceoperator-system/resources/flexibleserver-postgres.yaml:14]: hmm this looks like a generic resource, it probably needs an environment variable being added into the environments post build setup.

e.g. in here:
https://github.com/hmcts/cnp-flux-config/blob/master/apps/sscs/base/kustomize.yaml
adding an APPLICATION_TAG in there

File: apps/sscs/preview/aso/sscs-postgres.yaml [hunk 2]
```diff
@@ -6,6 +6,12 @@ metadata:
  annotations:
   serviceoperator.azure.com/reconcile-policy: detach-on-delete
 spec:
+ tags:
```
Comment (timja) [inline apps/sscs/preview/aso/sscs-postgres.yaml:9]: why do we have the generic version in apps/azureserviceoperator-system/resources/flexibleserver-postgres.yaml and then this one here?

File: apps/azureserviceoperator-system/resources/flexibleserver-postgres.yaml (hunk 1 above)
Comment (cpareek) [reply apps/azureserviceoperator-system/resources/flexibleserver-postgres.yaml]: I have added this

File: apps/sscs/preview/aso/sscs-postgres.yaml (hunk 2 above)
Comment (cpareek) [reply apps/sscs/preview/aso/sscs-postgres.yaml:9]: I think the ASO resource will need to setup in the apps/azureserviceoperator-system/resources first and then in the environement/app you want it.  Again, followed whats been done on service bus

Comment (timja) [reply apps/sscs/preview/aso/sscs-postgres.yaml:9]: i don't think that's right, sscs should just add the resource to their preview kustomize file and then they will get this.

Comment (timja) [reply apps/sscs/preview/aso/sscs-postgres.yaml:9]: although people probably want a bit of configuration available? i.e. database version

**Pull request closed without merging**